tests/Saver/MongoTest.php

```diff
@@ -17,9 +17,24 @@ public function testSave(): void
         $collection = $this->getMockBuilder(MongoCollection::class)
             ->disableOriginalConstructor()
             ->getMock();
-        $collection->expects($this->exactly(count($data)))
+
+        $collection
+            ->expects($this->exactly(count($data)))
             ->method('insert')
-            ->withConsecutive($this->equalTo($data));
+            ->withConsecutive(...array_map(function () {
+                    return [
+                        $this->callback(function ($data) {
```

Comment:
i don't understand how `$this->callback` is able to obtain `$data`, but it works correctly